Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5515,6 +5516,11 @@ std::optional<LayoutUnit> RenderBox::explicitIntrinsicInnerWidth() const
     if (style().containIntrinsicWidthType() == ContainIntrinsicSizeType::None)
         return std::nullopt;
 
+    if (element() && style().containIntrinsicWidthType() == ContainIntrinsicSizeType::AutoAndLength && shouldSkipContent()) {
```

Comment:
CIS auto needs to call  `shouldSkipContent()`, so it depends on `content-visibility`

Reply:
See https://w3c.github.io/csswg-drafts/css-sizing-4/#valdef-contain-intrinsic-width-auto-length